sessions/react-global-state/demo-start/components/Footer.js

```diff
@@ -0,0 +1,10 @@
+const dragonCount = "???";
+
+export default function Footer() {
```

Comment:
It might be easier to understand when we replace a local variable within the function with a prop from the parameters

```suggestion
export default function Footer() {
  const dragonCount = "???";
```